CHANGELOG_PENDING.md

```diff
@@ -1,3 +1,5 @@
 ### Improvements
 
 ### Bug Fixes
+
+- Fix #534: add StackReferenceArgs.builder() and StackReferenceArgs.Builder, deprecate StackReferenceArgs.getName()
```

Comment:
This sounds like an Improvement not a Bug Fix.

Reply:
thank you, moved